src/node.cc

```diff
@@ -1597,6 +1618,7 @@ static void Cwd(const FunctionCallbackInfo<Value>& args) {
 
 static void Umask(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
+  CHECK(env->is_main_instance());
```

Comment:
I think it should be safe to call `process.umask()` from a worker, just not `process.umask(mask)` (although you could argue it's not safe for the main isolate to change the umask under a worker's feet either.)